src/index.js

```diff
@@ -0,0 +1,24 @@
+import express from 'express';
+import BodyParser from 'body-parser';
+import mongoose from 'mongoose';
+import MovieRoutes from '../routes/MovieRoutes';
+
+// new instance of express and routes
+const app = express();
+
+// Connect to MongoDB
+mongoose.Promise = global.Promise;
+mongoose.connect('mongodb://localhost/movie-list');
+
+app.use(BodyParser.json());
+app.use(MovieRoutes);
+app.use((err, request, response) => response.status(500).send('Request error! ' + err));
```

Comment:
This error middleware should have 4 arguments, this wont actually catch our errors!